MonoGame.Framework/Graphics/Effect/IEffectFog.cs

```diff
@@ -48,32 +10,31 @@ namespace Microsoft.Xna.Framework.Graphics
 	public interface IEffectFog
 	{
         /// <summary>
-        /// The floating point fog color.
+        /// Gets or set the fog color in floating point format.

```

Comment:
It is useful when you get a read-only property.  This is common in Microsoft documentation where a property might be described as "Gets" or "Gets and sets" to differentiate between read-only and read/write properties.